semantic-core/src/Analysis/ScopeGraph.hs

```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE FlexibleContexts, OverloadedStrings, RankNTypes, RecordWildCards, TypeApplications, TypeOperators #-}
 module Analysis.ScopeGraph
 ( ScopeGraph(..)
+, Ref (..)
```

Comment:
How come?

Reply:
I needed the `Ref` constructor visible so I could turn it into JSON in the `Instances` module.